keylime/tpm/tpm_abstract.py

```diff
@@ -66,13 +69,19 @@ def readPolicy(configval):
 
 class AbstractTPM(metaclass=ABCMeta):
     # Abstract base class
-    EXIT_SUCESS = 0
-    TPM_IO_ERR = 5
-    EMPTYMASK = "1"
-    MAX_NONCE_SIZE = 64
+    EXIT_SUCESS: int = 0
+    TPM_IO_ERR: int = 5
+    EMPTYMASK: str = "1"
+    MAX_NONCE_SIZE: int = 64
+
+    need_hw_tpm: bool
+    global_tpmdata: Optional[Dict[str, Any]]
```

Comment:
Is this `Any` OK, can we force `str` or `bytes`? Do we have other types? Dates maybe?

Reply:
Right... `bytes` is the right datatype.

Reply:
Actually the datatypes has to be `Union[str, int]` since we have key handles in there and sometime filenames etc.